canonical/ubuntu.com
# WD-12060 - rebrand /cloud/hybrid-cloud

## Done

Rebrand /cloud/hybrid-cloud. No content was changed, layout only

## QA

- [demo link](https://ubuntu-com-14096.demos.haus/cloud/hybrid-cloud)
- [figma](https://www.figma.com/design/NpI2wzW86XXPWxsSeomYfy/24.10-u.c%2Fcloud?node-id=27-6057&t=z17uCr16KZkDDQRN-0)
- View the site on demo
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correct according to figma

## Issue / Card
[WD-12060](https://warthogs.atlassian.net/browse/WD-12060)

[WD-12060]: https://warthogs.atlassian.net/browse/WD-12060?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
